packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -283,22 +283,31 @@ const FilterConfiguration: FC<Props> = ({
                         </Tooltip>
                     )}
 
-                <Button
-                    size="xs"
-                    variant="filled"
-                    className={Classes.POPOVER2_DISMISS}
-                    disabled={
-                        !isFilterConfigurationApplyButtonEnabled(
-                            draftFilterRule,
-                        )
-                    }
-                    onClick={() => {
-                        setSelectedTabId(FilterTabs.SETTINGS);
-                        if (!!draftFilterRule) onSave(draftFilterRule);
-                    }}
+                <Tooltip
+                    label="Filter field and value required"
+                    disabled={isFilterConfigurationApplyButtonEnabled(
+                        draftFilterRule,
+                    )}
                 >
-                    Apply
-                </Button>
+                    <Box>
```

Comment:
we need to wrap in `Box` in case the button is `disabled` - that would mean the tooltip events wouldn't work right?

Reply:
Yea, exactly. I don't know why they still have it that way, but this seems to work.